html/php-pre-challenge3/index.php

```diff
@@ -1,6 +1,47 @@
 <?php
 $limit = $_GET['target'];
+if (is_string($limit) == false || $limit < 1 || preg_match('/^([1-9]\d*|0)\.(\d+)?$/', $limit) ) {
```

Comment:
細かいですが厳密比較 `===` に統一しましょう
```suggestion
if (is_string($limit) === false || $limit < 1 || preg_match('/^([1-9]\d*|0)\.(\d+)?$/', $limit) ) {
```